docs/bcfusion/validators/staking.md

```diff
@@ -0,0 +1,133 @@
+commission rate and the slashing rate of the validator.
+
+### Redelegate
+
+To redelegate BNB from one validator to another, a delegator needs to send a `Redelegate` transaction to the BNB smart
+chain, specifying the following information:
+
+- **Source operator address**: The BEP20 address of the source validator, which will send the BNB to the destination
+  validator.
+- **Destination operator address**: The BEP20 address of the destination validator, which will receive the BNB from the
+  source validator.
+- **Amount**: The amount of BNB that the delegator wants to redelegate from the source validator to the destination
+  validator.
+- **Delegate Voting Power**: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the destination validator for governance.
+
+The `Redelegate` transaction will deduct the amount of staking credit from the source validator and issue the
+corresponding staking credit to the destination validator. The delegator will then share the rewards and the risks with
+the destination validator, according to the commission rate and the slashing rate of the destination validator.
+The `Redelegate` transaction does not incur the unbonding period, but it will incur the redelegation fee,
+which is designed to prevent delegators from frequently switching between validators to chase
+the highest rewards or avoid the highest risks.
+
+### Undelegate
+
+To undelegate BNB from a validator, a delegator needs to send an `Undelegate` transaction to the BNB smart chain,
+specifying the following information:
+
+- **Operator address**: The BEP20 address of the validator, which will send the BNB to the delegator.
+- **Amount**: The amount of BNB that the delegator wants to unstake from the validator.
+
+The `Undelegate` transaction will deduct the amount of staking credit from the validator and return the corresponding
+BNB to the delegator address. However, the BNB will be locked for a period of time, called the **unbonding period**,
+before the delegator can use it. The unbonding period is currently set to 7 days, and it is designed to prevent
+delegators from quickly withdrawing their BNB in case of a validator misbehavior or a network attack.
+
+### Claim
+
+To claim the unbond BNB and the rewards, a delegator should send a `Claim` transaction to the BNB smar chain, specifying
+the following information:
+
+- **Delegator address**: The BEP20 address of the delegator, which will receive the rewards from the validator.
+- **Queued unbond number**: The number of unbond requests to be claimed, and 0 means claim BNB and rewards from
+  all the unbond requests.
+
+The `Claim` transaction will return the delegated BNB and rewards to the delegator. Be noted, a delegator can only get
+the rewards after unbond. Before undelegation, the reward will be furthur staked to boost a delegator's income.
```

Comment:
Can we have a slash.md to explain the three kind of slash?

Reply:
added